core/src/test/scala/unit/kafka/zk/EmbeddedZookeeper.scala

```diff
@@ -40,7 +48,8 @@ class EmbeddedZookeeper() extends Logging {
   val logDir = TestUtils.tempDir()
   val tickTime = 800 // allow a maxSessionTimeout of 20 * 800ms = 16 secs
 
-  System.setProperty("zookeeper.forceSync", "no")  //disable fsync to ZK txn log in tests to avoid timeout
+  System.setProperty("zookeeper.forceSync", "no")  // disable fsync to ZK txn log in tests to avoid timeout
```

Comment:
optional nit: I personally prefer to leave this kind of styling changes as in the upstream.  This makes future rebase easier.